src/calendar-app.ts

```diff
@@ -0,0 +1,774 @@
+		if (isDesktop()) {
+			import("./common/native/main/UpdatePrompt.js").then(({ registerForUpdates }) => registerForUpdates(locator.desktopSettingsFacade))
+		}
+
+		const userLanguage = deviceConfig.getLanguage() && languages.find((l) => l.code === deviceConfig.getLanguage())
+
+		if (userLanguage) {
+			const language = {
+				code: userLanguage.code,
+				languageTag: languageCodeToTag(userLanguage.code),
+			}
+			lang.setLanguage(language).catch((e) => {
+				console.error("Failed to fetch translation: " + userLanguage.code, e)
+			})
+
+			if (isDesktop()) {
+				locator.desktopSettingsFacade.changeLanguage(language.code, language.languageTag)
+			}
+		}
+
+		locator.logins.addPostLoginAction(() => locator.postLoginActions())
+
+		if (isOfflineStorageAvailable()) {
+			const { CachePostLoginAction } = await import("./common/offline/CachePostLoginAction")
+			locator.logins.addPostLoginAction(
+				async () =>
+					new CachePostLoginAction(
+						await locator.calendarModel(),
+						locator.entityClient,
+						locator.progressTracker,
+						locator.cacheStorage,
+						locator.logins,
+					),
+			)
+		}
+
+		styles.init()
+
+		const contactViewResolver = makeViewResolver<
+			ContactViewAttrs,
+			ContactView,
+			{
+				drawerAttrsFactory: () => DrawerMenuAttrs
+				header: AppHeaderAttrs
+				contactViewModel: ContactViewModel
+				contactListViewModel: ContactListViewModel
+			}
+		>(
+			{
+				prepareRoute: async () => {
+					const { ContactView } = await import("./mail-app/contacts/view/ContactView.js")
+					const drawerAttrsFactory = await locator.drawerAttrsFactory()
+					return {
+						component: ContactView,
+						cache: {
+							drawerAttrsFactory,
+							header: await locator.appHeaderAttrs(),
+							contactViewModel: await locator.contactViewModel(),
+							contactListViewModel: await locator.contactListViewModel(),
+						},
+					}
+				},
+				prepareAttrs: (cache) => ({
+					drawerAttrs: cache.drawerAttrsFactory(),
+					header: cache.header,
+					contactViewModel: cache.contactViewModel,
+					contactListViewModel: cache.contactListViewModel,
+				}),
+			},
+			locator.logins,
+		)
+
+		const paths = applicationPaths({
+			login: makeViewResolver<LoginViewAttrs, LoginView, { makeViewModel: () => LoginViewModel }>(
+				{
+					prepareRoute: async () => {
+						const migrator = await locator.credentialFormatMigrator()
+						await migrator.migrate()
+
+						const { LoginView } = await import("./common/login/LoginView.js")
+						const makeViewModel = await locator.loginViewModelFactory()
+						return {
+							component: LoginView,
+							cache: {
+								makeViewModel,
+							},
+						}
+					},
+					prepareAttrs: ({ makeViewModel }) => ({ targetPath: "/mail", makeViewModel }),
+					requireLogin: false,
+				},
+				locator.logins,
+			),
+			termination: makeViewResolver<TerminationViewAttrs, TerminationView, { makeViewModel: () => TerminationViewModel; header: AppHeaderAttrs }>(
```

Comment:
we don't need termination either and some other paths below

Reply:
I would argue we do, you are able to create an account with this app. Thus, you should be able terminate the account with the app.

Reply:
termination is a very specific case, it is accessible only via the website and calendar app is not running as a webapp